### PR TITLE
Update configuration with dataset

### PR DESCRIPTION
**Pull request by KodiaqQ**
- Fixed configuration with `scale_estimation`.

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

File: llm_bench/python/utils/nncf_utils.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ def get_compressed_path(output_dir: str, base_precision, option: str):
 
 
 INT4_MODEL_CONFIGURATION = {
-    "dolly-v2-3b": {"mode": nncf.CompressWeightsMode.INT4_ASYM, "group_size": 128, "ratio": 1.0, "scale": True},
+    "dolly-v2-3b": {"mode": nncf.CompressWeightsMode.INT4_ASYM, "group_size": 128, "ratio": 1.0,
+                    "dataset": {"name": "wikitext,wikitext-2-v1,train[:1000],text", "scale_estimation": True}},
```
[the remark this answers is not in the document]
Comment (KodiaqQ) [reply llm_bench/python/utils/nncf_utils.py:42]: Yes, I've used this number of samples according to the default configuration from changes before this PR.
Maybe @andreyanufr, can comment this number of the samples.

**Comment by KodiaqQ** [comment]
Here is the confirmation of configuration applicability.
image
cc @nikita-savelyevv

**Comment by nikita-savelyevv** [comment]
@KodiaqQ Could you please try compress from model_id? I've just tried it out and I see that scale estimation is not applied.

**Comment by KodiaqQ** [comment]
> @KodiaqQ Could you please try compress from model_id? I've just tried it out and I see that scale estimation is not applied.

It might be a problem with your conversion command line since convert.py is very sensitive in this - [helpers.py](https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/utils/conversion_utils/helpers.py#L198)
Here is my result:
image

**Comment by nikita-savelyevv** [comment]
> > @KodiaqQ Could you please try compress from model_id? I've just tried it out and I see that scale estimation is not applied.
> 
> It might be a problem with your conversion command line since convert.py is very sensitive in this - [helpers.py](https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/utils/conversion_utils/helpers.py#L198) Here is my result: image

What I've meant is to provide `databricks/dolly-v2-3b` instead of a path as `--model_id`. In this case I don't see scale estimation applied.

This is actually because in this case the config is imported from optimum-intel: https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/utils/conversion_utils/helpers.py#L192

**Comment by KodiaqQ** [comment]
> This is actually because in this case the config is imported from optimum-intel: https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/utils/conversion_utils/helpers.py#L192

This is not an issue for this PR. This is a general issue for llm_bench/python, as I can see. Am I right?

**Pull request closed without merging**